src/madpack/madpack.py

```diff
@@ -1409,16 +1419,41 @@ def main(argv):
         else:
             maddir_conf = maddir + "/config"
 
-        global maddir_lib
-        if portid == 'greenplum' and \
-           os.path.islink(maddir + "/../../../lib/postgresql/libmadlib.so"):
-           maddir_lib = '$libdir/libmadlib.so'
-        elif os.path.isfile(maddir + "/ports/" + portid + "/" + dbver +
+        global madlib_library_path
+        # Local build at ~/workspace/madlib/build/
+        if os.path.isfile(maddir_abs + "/src/ports/" + portid + "/" + dbver +
```

Comment:
The `If` part is under the maddir_abs folder but the `elif` part is outside (since we want to make sure we are looking at the Current folder, not Versions). This makes glob complicated because we don't want to search outside the madlib folder accidentally.

Reply:
The If part is under the maddir_abs folder but the elif part is outside (since we want to make sure we are looking at the Current folder, not Versions). This makes glob complicated because we don't want to search outside the madlib folder accidentally.